doc/mgr/influx.rst

```diff
@@ -54,6 +54,8 @@ Additional optional configuration settings are:
 :port: InfluxDB server port.  Default 8086
 :ssl: Use https connection for InfluxDB server. Use "true" or "false". Default false
 :verify_ssl: Verify https cert for InfluxDB server. Use "true" or "false". Default true
+:timeout: InfluxDB client connection timeout.  Defaults to 2s.
```

Comment:
It's kind of problematic to have a default 5s interval, but a 2s timeout -- if a couple of endpoints are offline or slow then the sum of timeouts will prevent the module from keeping up with the intended interval.

For this to be somewhat robust, the multi-destination transmission probably needs to be happening in parallel, or the timeout needs to be much less than the interval